packages/js/core/src/algorithms/get-implementations.ts

```diff
@@ -23,7 +23,7 @@ export const getImplementations = Tracer.traceFunc(
     const addAllImplementationsFromImplementationsArray = (
       implementationsArray: readonly InterfaceImplementations<Uri>[],
       wrapperInterfaceUri: Uri
-    ) => {
+    ): Result<true, Error> => {
```

Comment:
Is there a need for this to be a Result if in the `ok` case you're returning the data and in the `error` case you're throwing?

Reply:
I went back and forth on how to implement this and I meant to return the error instead of throwing it